### DISCUSSION TITLE
how to save multiple pdf to database?

### DISCUSSION THREAD

**Discussion by abcnow** [agno-agi/agno#221] [General]
I tried to upload multiple pdf but every time the next pdf replace the content of the first one in the database.. I wonder how could I be able to upload multiple pdf and search them..?

**Comment by ashpreetbedi** [comment]
@abcnow is the PDF with the same name? they should all be saved? 

Which app/cookbook are you using?

**Reply by abcnow** [reply]
All pdfs have different names...and they all should be saved..
I didn't know there were cookbooks available.. any link you can share? Thanks in advance !
